src/EditorFeatures/DiagnosticsTestUtilities/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -122,6 +122,27 @@ public void TestSupportedDiagnosticsMessageHelpLinkUri()
 
                 foreach (var descriptor in diagnosticAnalyzer.SupportedDiagnostics)
                 {
+                    // These don't come up in UI.
+                    if (descriptor.DefaultSeverity == DiagnosticSeverity.Hidden && descriptor.CustomTags.Contains(WellKnownDiagnosticTags.NotConfigurable))
+                    {
+                        continue;
+                    }
+
+                    if (descriptor.Id is "RE0001") // Currently not documented. https://github.com/dotnet/roslyn/issues/48530
+                    {
+                        continue;
+                    }
+
+                    if (descriptor.Id == "IDE0043") // Intentionally undocumented. It will be removed in favor of CA2241
+                    {
+                        continue;
+                    }
+
+                    if (descriptor.Id is "IDE1007" or "IDE1008")
```

Comment:
💡 File a follow-up issue to link documentation for IDE1007?

Reply:
@mavasani Do you know if this one is intentionally undocumented or is missing documentation?